ui/controls.js

```diff
@@ -809,6 +819,11 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
       return;
     }
 
+    // If you were fullscreen, leave fullscreen first.
```

Comment:
Should this be duplicated in two places?  It seems like it is an invariant you could put at the top of togglePip instead.

Reply:
Done!